sigmastate/src/main/scala/sigmastate/utxo/JitCostTable.scala

```diff
@@ -0,0 +1,407 @@
+package sigmastate.utxo
+
+import org.ergoplatform.SigmaConstants
+import sigmastate.{Downcast, Upcast}
+import sigmastate.lang.SigmaParser
+import sigmastate.lang.Terms.OperationId
+
+case class JitCostTable(operCosts: Map[OperationId, Int]) extends (OperationId => Int) {
```

Comment:
To reviewer:
This is version is mostly  the same as the old CostTable, but the plan is to make completely new Cost Model which better reflect JIT costs. 
The main invariant which will be guaranteed is that always JIT cost < AOT cost for all non spam scripts.
This shouldn't be hard to do since existing op costs are very conservative and new costs may be reduced proportional to new JIT interpreter speedup. (e.g. 2-3x)